torch/fx/experimental/GraphManipulation.py

```diff
@@ -61,18 +69,157 @@ def get_size_of_node(fx_module: GraphModule, node: Node) -> size_bytes:
+        nodes: [NODE],
+        weights {qualified_name: WEIGHT},
+    }
+    NODE
+    {
+        shape: [],
+        dtype: dtype,
+        target: target,
+        op_code: op_code,
+        name: name,
+        args: [],
+        kwargs: {}
+    }
+    WEIGHT
+    {
+        dtype: dtype,
+        is_quantized: bool,
+        shape: [],
+        quantization_info: QUANTIZATION
+    }
+    QUANTIZATION
+    {
+        qsheme: sqcheme,
+        q_scale: float,
+        q_zero_point: float,
+        q_per_channel_scales, [],
+        q_per_channel_zero_points: [],
+        q_per_channel_axis, int
+    }
+    """
+    serialized_dict = {}
+    serialized_dict["modules"] = {}
+    serialized_dict["weights"] = {}
+    serialized_dict["nodes"] = []
+    parameters = fx_module.named_parameters()
+    for name, p in parameters:
+        if isinstance(p, torch.Tensor):
+            weight = serialize_weight(p)
+            prefix = f"{name_prefix}." if name_prefix else ""
+            serialized_dict["weights"][prefix + name] = weight
+            weights[prefix + name] = p
+    for node in fx_module.graph.nodes:
+        node_rep = {}
+        # Get shape/type info, currently not needed for call_module.
+        if node.op != "call_module":
+            shape = getattr(node, "shape", None)
+            if shape:
+                node_rep["shape"] = serialize_shape(shape)
+            else:
+                raise RuntimeError(
+                    "Node has no shape attr, this is likely because shape propigation has not been run on this Graph."
+                )
+            dtype = getattr(node, "dtype", None)
+            if dtype:
+                node_rep["dtype"] = str(dtype)
+            else:
+                raise RuntimeError(
+                    "Node has no dtype attr, this is likely because shape propigation has not been run on this Graph."
+                )
+
+        # Recurse down into any submodules we are calling.
+        if node.op == "call_module":
+            submodules = dict(fx_module.named_modules())
+            if isinstance(submodules[node.target], torch.fx.GraphModule):
+                serialized_module = serialize_module(
+                    getattr(fx_module, node.target), weights, node.target
+                )
+                serialized_dict["modules"][node.target] = serialized_module
+
+        if node.op == "call_function":
+            node_rep["target"] = get_qualified_name(node.target)
+        else:
+            node_rep["target"] = str(node.target)
+
+        # Make sure we capture all constants.
+        if node.op == "get_attr":
+            target = getattr(fx_module, node.target)
+            prefix = f"{name_prefix}." if name_prefix else ""
+            qualname = prefix + node.target
+            if isinstance(target, torch.Tensor) and qualname not in weights:
+                weight = serialize_weight(target)
+                prefix = f"{name_prefix}." if name_prefix else ""
```

Comment:
This also exists higher up too (lines 203 and 164) -- probably makes sense to remove all of them that are in deeper scopes and just set this once.